src/test/java/nl/tudelft/ewi/devhub/webtests/views/AssignmentView.java

```diff
@@ -0,0 +1,59 @@
+package nl.tudelft.ewi.devhub.webtests.views;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Getter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Douwe Koopmans on 4-5-16.
+ */
+public class AssignmentView extends ProjectSidebarView {
+
+    private static final By RECENT_ASSIGNMENTS_HEADER = By.xpath("//h4[starts-with(normalize-space(.), 'Assignments')]");
+    public AssignmentView(WebDriver driver) {
+        super(driver);
+    }
+    
+    @Override
+    protected void invariant() {
+        super.invariant();
+        assertNotNull(getDriver().findElement(RECENT_ASSIGNMENTS_HEADER));
+        assertTrue(currentPathStartsWith("/courses/ti1705/TI1705/groups/"));
+    }
+
+    public Assignment getAssignment() throws ParseException {
+        String author;
+        Date date;
+        WebElement submit = getDriver().findElement(By.cssSelector("a[href^=\"deliveries\"]"));
+
+        final List<WebElement> elements = getDriver().findElements(By.tagName("dd"));
+        author = elements.get(0).getText();
```

Comment:
Please also initialize the variable here as well. Otherwise it is a non-constant variable.